Detectors/Base/include/DetectorsBase/GRPObject.h

```diff
@@ -0,0 +1,132 @@
+/// \author M. Al-Turany, S. Wenzel - June 2014
+
+#ifndef ALICEO2_DATA_GRPOBJECT_H_
+#define ALICEO2_DATA_GRPOBJECT_H_
+
+#include <cstdint>
+#include <Rtypes.h>
+#include <DetectorsBase/DetID.h>
+
+namespace o2 {
+namespace Data {
+
+  
+class GRPObject {
+
+public:
+
+  GRPObject() = default;
+  ~GRPObject() = default;
+    
+  time_t getTimeStart() const { return mTimeStart; }
+  time_t getTimeEnd() const { return mTimeEnd; }
+  void setTimeStart(time_t t) { mTimeStart = t; }
+  void setTimeEnd(time_t t) { mTimeEnd = t; }
+
+  float getCrossingAngle() const { return mCrossingAngle; }
+  void  setCrossingAngle(float v) { mCrossingAngle = v; }
+  
+  int getBeamAZ(int b) const { return mBeamAZ[b]; }
+  int getBeamZ(int b) const { return mBeamAZ[b] % 1000; }
+  int getBeamA(int b) const { return mBeamAZ[b] / 1000; }
+  float getBeamZ2A(int b) const;
+  void setBeamAZ(int az, int beam) { mBeamAZ[beam] = az; }
+  void setBeamAZ(int a, int z, int beam) { setBeamAZ(a * 1000 + z, beam); }
+
+  void  setBeamEnergyPerZ(float v) { mBeamEnergyPerZ = v; }
+  float getBeamEnergyPerZ() const { return mBeamEnergyPerZ; }
+  float getBeamEnergyPerNucleon(int beam) const { return mBeamEnergyPerZ * getBeamZ2A(beam); }
+  
+  float getSqrtS() const;
+  
+  float getL3Current() const { return mL3Current; }
+  float getDipoleCurrent() const { return mDipoleCurrent; }
+  short getL3Polarity() const { return mL3Polarity; }
+  short getDipolePolarity() const { return mDipolePolarity; }
+  void  setL3Current(float v)  { mL3Current = v; }
+  void  setDipoleCurrent(float v) { mDipoleCurrent = v; }
+  void  setL3Polarity(short v) { mL3Polarity = v; }
+  void  setDipolePolarity(short v) { mDipolePolarity = v; }  
+  
+  const std::string& getDataPeriod() const { return mDataPeriod; }
+  void setDataPeriod(const char* v) { mDataPeriod = v; }
+  
+  const std::string& getLHCState() const { return mLHCState; }
+  void setLHCState(const char* v) { mLHCState = v; }
+
+  void setRun(int r) { mRun = r; }
+  int  getRun() const { return mRun; }
+
+  void setFill(int f) { mFill = f; }
+  int  getFill() const { return mFill; }
+
+  std::int32_t getDetsReadOut() const { return mDetsReadout; }
+  std::int32_t getDetsTrigger() const { return mDetsTrigger; }
+
+  void setDetsReadOut(std::int32_t mask) { mDetsReadout = mask; }
+  void setDetsTrigger(std::int32_t mask) { mDetsTrigger = mask; }
+
+  void addDetReadOut(o2::Base::DetID id) { mDetsReadout |= id.getMask(); }
+  void remDetReadOut(o2::Base::DetID id) { mDetsReadout &= ~id.getMask(); }
+
+  void addDetTrigger(o2::Base::DetID id) { mDetsTrigger |= id.getMask(); }
+  void remDetTrigger(o2::Base::DetID id) { mDetsTrigger &= ~id.getMask(); }
+
+  bool isDetReadOut(o2::Base::DetID id) const { return (mDetsReadout&id.getMask())!=0; }
+  bool isDetTriggers(o2::Base::DetID id) const { return (mDetsTrigger&id.getMask())!=0; }
+  
+  void print() const;
+  
+private:
+  static constexpr double ProtonMass = 0.938272;
```

Comment:
OK, to avoid multiple redefinition of same constants by different authors, I've created Common/Constants directory where we can dump the headers with different constant categories. At the moment just PhysicsConstants.h and LHCConstants.h (will move there it separate PR some LHC constants already defined elsewhere).